scripts/bootstrap

```diff
@@ -2,7 +2,7 @@
 set -e
 
 if [[ $(uname -s) == 'Darwin' ]]; then 
-	echo "Please make sure you have helm and yq installed"
+	echo "Please make sure you have helm and yq (v3 or higher) installed"
```

Comment:
```suggestion
	echo "Please make sure you have helm v3 and yq installed"
```

Reply:
Changed to `Please make sure you have helm v3 and yq v3 installed`